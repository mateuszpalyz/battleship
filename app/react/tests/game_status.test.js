import React from 'react';
import GameStatus from '../components/game_status';
import renderer from 'react-test-renderer';

test('Component displays waiting info for waiting status', () => {
  const component = renderer.create(
    <GameStatus status='waiting'/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('Component displays started info for started status', () => {
  const component = renderer.create(
    <GameStatus status='started'/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
