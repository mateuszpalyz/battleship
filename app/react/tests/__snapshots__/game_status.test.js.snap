// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component displays started info for started status 1`] = `
<div
  className="game-status"
>
  <p>
    Game Status: 
    <span>
      fight arrr!!!
    </span>
  </p>
</div>
`;

exports[`Component displays waiting info for waiting status 1`] = `
<div
  className="game-status"
>
  <p>
    Game Status: 
    <span>
      waiting for second player
    </span>
  </p>
</div>
`;
